packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -236,6 +242,13 @@ export interface SubnetSelection {
    */
   readonly onePerAz?: boolean;
 
+  /**
+   * List of provided subnet filters.
+   *
+   * @default - none
+   */
+  readonly subnetFilters?: ISubnetSelector[];
```

Comment:
The name of the field and the name of the type should probably be the same here.

I guess call the class `SubnetFilter` as well then.

Reply:
Done